### PR TITLE
Switch from WQP to NLDI

### PR DESCRIPTION
**Pull request by EthanGrahn**
Modified the jenkins file to grab secrets from NLDI instead of WQP. Removed `POSTGRES_PASSWORD` since it was never used.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -65,7 +65,6 @@ pipeline {
           env.WMADATA_SCHEMA_OWNER_PASSWORD = secretsJson.WMADATA_SCHEMA_OWNER_PASSWORD
           env.WMADATA_DB_READ_ONLY_USERNAME = secretsJson.WMADATA_DB_READ_ONLY_USERNAME
           env.WMADATA_DB_READ_ONLY_PASSWORD = secretsJson.WMADATA_DB_READ_ONLY_PASSWORD
-          env.POSTGRES_PASSWORD = secretsJson.POSTGRES_PASSWORD
```
Comment (mteather-usgs) [inline Jenkinsfile:68]: POSTGRES_PASSWORD is used in z1_postgres_liquibase.sh.

**Review by mteather-usgs — CHANGES_REQUESTED** [review]
Think we need POSTGRES_PASSWORD because it is used when we execute the script z1_postgres_liquibase.sh on line 76.

File: Jenkinsfile (hunk 1 above)
Comment (EthanGrahn) [reply Jenkinsfile]: Added back in with new commit https://github.com/NWQMC/wmadata-schema-core/pull/5/commits/9e4a9c25be43587bb724d74287b13bf3abd9b5cc

**Review by mteather-usgs — APPROVED** [review]
Perfect, thanks!

**Pull request merged**